chaos-bugbounty-list.json

```diff
@@ -7467,6 +7467,65 @@
 	      "lunarly.com",
 	      "mother-earthproducts.co"
       ]
+    },
+    {
+      "name": "Tim Brasil",
+      "url": "https://admin.bughunt.com.br/program/detail?6058de165576f62d921fc844",
+      "bounty": true,
+      "domains": [
+        "site.tim.com.br"
+      ]
+    },
+    {
+      "name": "BitcoinTrade Bug Bounty Program",
+      "url": "https://admin.bughunt.com.br/program/detail?5f98855f51ec4c09107c0534",
+      "bounty": true,
+      "domains": [
+        "api.bitcointrade.com.br",
+        "bitcointrade.com.br",
+        "bitcointrade.com.br",
+        "brocker.bitcointrade.com.br",
+        "apigw.bitcointrade.com.br",
+        "core.bitcointrade.com.br",
+        "coreuat.bitcointrade.com.br",
+        "bitcoinback.com.br",
+        "backofficespa.bitcointrade.com.br",
+        "cdn.bitcoinback.com.br",
+        "btcback.bitcointrade.com.br",
+        "apistg.bitcointrade.com.br",
+        "link.bitcointrade.com.br",
+        "www.bitcoinback.com.br"
+      ]
+    },
+    {
+      "name": "OLX Brasil Bug Bounty",
+      "url": "https://admin.bughunt.com.br/program/detail?60ca003869707962894a5b33",
+      "bounty": true,
+      "domains": [
+        "site.tim.com.br",
+        "https://play.google.com/store/apps/details?id=com.schibsted.bomnegocio.androidApp&hl=pt_BR&showAllReviews=true",
```

Comment:
These should not be included, see also: https://github.com/projectdiscovery/public-bugbounty-programs#-notes